pkg/odo/cli/url/create.go

```diff
@@ -100,25 +100,27 @@ func (o *URLCreateOptions) Complete(name string, cmd *cobra.Command, args []stri
 		o.Context = genericclioptions.NewContext(cmd)
 	}
 
-	o.Client = genericclioptions.Client(cmd)
+	if experimental.IsExperimentalModeEnabled() && util.CheckPathExists(o.DevfilePath) {
```

Comment:
IMO, there is no need of additional if 
`if experimental.IsExperimentalModeEnabled() && util.CheckPathExists(o.DevfilePath) && !pushtarget.IsPushTargetDocker()`

Reply:
@adisky It's required because there's code in that block farther down that's required when the pushtarget is set to Docker, and using `if experimental.IsExperimentalModeEnabled() && util.CheckPathExists(o.DevfilePath) && !pushtarget.IsPushTargetDocker()` would prevent it from executing.